frontend/components/arts/ArtistCard.vue

```diff
@@ -80,34 +82,49 @@ export default {
     }
   },
   computed: {
+    // Here's a good read on when to use computed() -> https://medium.com/notonlycss/the-difference-between-computed-and-methods-in-vue-js-9cb05c59ed98
     mediaData() {
+      // This is an exact copy of the `media` Props in line 64, it just has a different variable name
+      // We're not preprocessing the `media` Props in any way, so this shouldn't be under computed
+      // I suggest we get rid of this sort of pattern.
       return this.media
     },
     artists() {
+      // This, I think, is a good usage. However, I believe it shouldn't 
+      // be dependent on another computed variable. You could go with
+      // return this.media.properties.artists
       return this.mediaData.properties.artists
     },
     mediaType() {
+      // Same comment for artists.
       return this.mediaData.file_type
     },
     videoThumbnail() {
+      // Good usage. Just don't use mediaData.
       return `https://img.youtube.com/vi/${this.getYoutubeVideoID(
         this.mediaData.url
       )}/hqdefault.jpg`
     },
     isLayoutTile() {
+      // Nice
       return this.layout !== 'landscape'
     },
     mediaExist() {
+      // Nice -> Could be renamed to mediaExists. With 's' as it is singular.
       return this.mediaData.media_file !== null
     },
     artImage() {
+      // Good usage. Would be better if this had a comment. It's not readable, so a helptext would be great.
       return this.mediaExist
         ? this.mediaData.media_file
         : this.mediaType === 'video'
         ? this.videoThumbnail
         : require('@/assets/images/public_art_icon.svg')
```

Comment:
How do you guys feel about ternary ops versus if {} else {} blocks?

I'd agree ternary ops have some advantage when returning a value

Reply:
For me, if it's simple, I'd use the ternary operator. If it's not, I'd use an if-else block.